src/test/java/tests/ForgotPasswordTests.java

```diff
@@ -0,0 +1,41 @@
+package tests;
+
+import helpers.DataHelper;
+import helpers.LogHelper;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import page_objects.ForgotPasswordPage;
+import page_objects.LoginPage;
+import page_objects.RegisterPage;
+
+public class ForgotPasswordTests extends BaseTests {
+    ForgotPasswordPage forgotPasswordPage = new ForgotPasswordPage();
+    LoginPage loginPage = new LoginPage();
+    RegisterPage registerPage = new RegisterPage();
+
+    private final String email = DataHelper.getRandomEmail();
+    private final String password = "123456789";
+    private final String pid = "123456789";
+
+    @BeforeMethod()
+    public void preCondition() {
+        registerPage.goToRegisterPage();
+        registerPage.register(email, password, password, pid);
+    }
+
+    @Test(description = "User can reset password successfully")
+    public void tc09_UserCanResetPassword() {
+        LogHelper.info("Go to Login page");
+        loginPage.goToLoginPage();
+
+        LogHelper.info("Click on 'Forgot password page' link");
+        forgotPasswordPage.clickForgotPasswordLink();
+
+        LogHelper.info("Reset password");
+        forgotPasswordPage.resetPassword(email);
+
+        LogHelper.info("Check the notification message");
+        Assert.fail("Instructions to reset your password have been sent to " + email);
```

Comment:
describe more specifics about the issue.

Reply:
I fixed it. Please help me to check it. Thank so much